### DISCUSSION TITLE
Admin-Dashboard Dev Login

### DISCUSSION THREAD

**Discussion by StackSurfer** [Bootstrap-Academy/Bootstrap-Academy#67] [Q&A] [answered]
Hello. I have a question regarding the Admin Dashboard. While exploring the projects, issues, and tickets for the first time, I encountered an issue with the Admin Dashboard.

Is there a specific development login for the Admin Dashboard that I might have missed? I looked through the READMEs and repositories, but did not find any information.

**Comment by Defelo** [comment, marked as answer]
Hi, at the moment there are no publicly known test admin credentials, so for now you can just create a regular account on the [test instance](https://bootstrap-academy-frontend.pages.dev/) and send me your username, so I can grant you access to the admin dashboard.

**Reply by StackSurfer** [reply]
Thanks for your fast reply. My Username on Dev is "Joggle"

**Reply by Defelo** [reply]
Ok, you should have access to the admin dashboard now
